src/V3EmitCFunc.cpp

```diff
@@ -422,6 +422,8 @@ void EmitCFunc::emitCCallArgs(const AstNodeCCall* nodep, const string& selfPoint
         if (comma) puts(", ");
         if (VN_IS(nodep->backp(), CAwait) || !nodep->funcp()->isCoroutine()) {
             puts("vlProcess");
+        } else if (inproc) {
```

Comment:
Does `inproc` mean `inProcess`? Let's spell it out, it's not too long, and easier to read.

How hard would it be to move this whole `if (nodep->funcp()->needProcess()) {` block to some earlier stage like `Timing`? I'm not 100% comfortable with `Emit` making these decisions.